.ci/build.sh

```diff
@@ -74,14 +74,24 @@ echo "----> Check out gh-pages branch"
 git clone --depth=1 "$GIT_REPO" --branch=gh-pages "$OUT_DIR"
 
 echo "----> Moving new charts into repo"
-mv -nv "${tmp}"/*.tgz "${OUT_DIR}/"
+mv -v "${tmp}"/*.tgz "${OUT_DIR}/"
 
 pushd "$OUT_DIR"
 new_charts="$(git status --short | grep '^??' --count ||:)"
+modified_charts="$(git status --short | grep '^.M' --count ||:)"
+
+if [ "$modified_charts" -ne 0 ]; then
```

Comment:
If I read correctly, this will only evaluate to `true` when charts have been modified?